src/main/java/com/iridium/iridiumskyblock/api/IridiumSkyblockAPI.java

```diff
@@ -47,7 +43,8 @@ private IridiumSkyblockAPI(@NotNull IridiumSkyblock iridiumSkyblock) {
      * @return the instance of this api
      * @since 3.0.0
      */
-    public static @NotNull IridiumSkyblockAPI getInstance() {
+    public static IridiumSkyblockAPI getInstance() {
+        if(instance==null && IridiumSkyblock.getInstance()!=null) instance = new IridiumSkyblockAPI(IridiumSkyblock.getInstance());
```

Comment:
I think it would be better if you used a multiline if statement here for better readability

```suggestion
        if(instance == null && IridiumSkyblock.getInstance() != null) {
            instance = new IridiumSkyblockAPI(IridiumSkyblock.getInstance());
        }
```

I also think the concerns about this potentially returning null weren't addressed?

Reply:
The comment above mentions it might return null during initialization.

Reply:
Ah, it was displaying the wrong Javadoc. How about an additional @Nullable for consistency with the other nullable API methods, though? It would also be more accessible

Reply:
Can you send it as suggestions? Because I'm on movile and can't add it.